### PR TITLE
Adding a view for the work retrieved by a search.  fixes #1109

### LINKED ISSUE #1109: Implement Catalog View of a work
We need to implement the Work catalog view.

Currently it is just a todo

### PR DESCRIPTION
**Pull request by cam156**

### CODE REVIEW

File: app/views/generic_works/_generic_work.html.erb [hunk 1]
```diff
@@ -1,2 +1,15 @@
-TODO: Placeholder for a generic work <br />
 <%= generic_work.title %>
```
Comment (mjgiarlo) [inline app/views/generic_works/_generic_work.html.erb]: Do we still want this title here too?

**Comment by mjgiarlo** [comment]
Would you mind including a screenshot?

File: spec/views/generic_work/_generic_work.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe 'generic_works/_generic_work.html.erb', :type => :view do
+  context 'displaying a generic_file' do
+    let(:generic_work) do
+      mock_model(SolrDocument,
+                  title: "work title", id: "123", depositor: "user1", creator: "igor",
```
Comment (mjgiarlo) [inline spec/views/generic_work/_generic_work.html.erb_spec.rb]: Glad you had fun with this one. :japanese_ogre:

**Comment by mjgiarlo** [comment]
:+1:

**Comment by cam156** [comment]
I tried to include a screen shot, but unfortunately the pcdm branch was broken yesterday.  All I get is an error message now.  If I get a chance on Thursday, which might not be over likely, I will try to include a screen shot then.

File: app/views/generic_works/_generic_work.html.erb (hunk 1 above)
Comment (cam156) [inline app/views/generic_works/_generic_work.html.erb]: Not sure.  I was using the same format we had for the GenericFile

**Comment by mjgiarlo** [comment]
OK, don't sweat it then, @cam156.

File: app/views/generic_works/_generic_work.html.erb (hunk 1 above)
Comment (mjgiarlo) [inline app/views/generic_works/_generic_work.html.erb]: That seems like a reasonable approach!

**Comment by mjgiarlo** [comment]
OK, :shipit:

**Comment by cam156** [comment]
Had to regenerate my test app.  Here is the screen shot...

screen shot 2015-05-19 at 3 32 46 pm

**Pull request merged**